openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -145,9 +143,25 @@ def collect_stats():
             'platform_url': platform_url,
             'statistics_level': 'enthusiast',
             'students_per_country': json.dumps(students_per_country),
-            'generated_certificates': json.dumps(generated_certificates),
-            'registered_students': json.dumps(registered_students),
-            'enthusiastic_students': json.dumps(enthusiastic_students),
         })
 
-    send_instance_statistics_to_acceptor(olga_acceptor_url, data)
+    result = send_instance_statistics_to_acceptor(olga_acceptor_url, data)
+    set_last_sent_date(result, access_token, last_dates)
+
+
+def set_last_sent_date(result, token, dates):
+    """
+    Set last sent dates to the cache.
+
+    :param result: result of the http client request that were sent a statistics to the OLGA
+    :param statistics_level: statistics level - may be "paranoid" or "enthusiast"
```

Comment:
The function 'set_last_sent_date' doesn't have this parameter. Please rewrite the docstring for actualization information.